Assets/PatchKit Patcher/Scripts/UI/BorderlessWindow.cs

```diff
@@ -142,7 +175,7 @@ private void Awake()
 #endif
         }
 
-        private void EnforceCorrectScreenSize()
+        private void EnforceCorrectScreenSize(float scaler)
```

Comment:
I'd recommend renaming argument to `float screenScale`.

Reply:
done